drivers/gpio/gpio_msp432p4xx.c

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright (c) 2022, Dhruva Gole
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ti_msp432p4xx_gpio
+#include <errno.h>
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/init.h>
+#include <zephyr/kernel.h>
+#include <zephyr/sys/sys_io.h>
+
+/* Driverlib includes */
+#include <inc/hw_types.h>
+#include <inc/hw_memmap.h>
+#include <inc/hw_ints.h>
+#include <inc/hw_gpio.h>
+#include <driverlib/rom.h>
+#include <driverlib/pin.h>
+
+#include "gpio_utils.h"
+
+static const uint8_t pinTable[] = {
```

Comment:
@vaishnavachath  can you help me with where these values are supposed to come from? What do they mean?